src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,28 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.NoSuchElementException;
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
```

Comment:
You should remove the import for `MovieDaoImpl` since it's not a good practice to reference a specific implementation in the service layer. The service should only be aware of the interface `MovieDao` to follow the Dependency Inversion principle.

Reply:
You should remove the import for MovieDaoImpl since it's not a good practice to reference a specific implementation in the service layer. The service should only be aware of the interface MovieDao to follow the Dependency Inversion principle.